config/routes.rb

```diff
@@ -39,7 +39,7 @@
     collection do
       get 'initialize_bookmark'
       delete 'clear'
-      match 'bulk_ris', to: 'bookmarks#bulk_ris', via: [:get, :post], as: 'bulk_ris_solr_document'
+      match 'bulk_ris/:item_ids', to: 'bookmarks#bulk_ris', via: [:get, :post], as: 'bulk_ris'
```

Comment:
I don't think `:item_ids` is needed here since we aren't using params in the path. I may not fully understand how this works though.

Reply:
Ah, sorry.  I understand now: myaccount does that.